src/types/crossChainAction.ts

```diff
@@ -73,6 +73,7 @@ interface KlimaStakingAction {
 	containsSwitchChain?: boolean;
 	receiveAmount?: string;
 	bridgeUsed?: string;
+  gasCost?: string;
```

Comment:
hmmm, why you're not using `ICrossChainAction` `estimated` for `gasCost`?

Reply:
that's because the estimated object has two other properties and the gasCost in estimated is in terms of Ethers but this is straight USD cost from offer